plugins/main/public/components/common/search-bar/use-search-bar.ts

```diff
@@ -126,6 +132,16 @@ const useSearchBarConfiguration = (
         : console.warn('setFilters function is not defined');
       props?.onFiltersUpdated && props?.onFiltersUpdated(userFilters);
     },
+    refreshInterval:
+      queryService.timefilter.timefilter.getRefreshInterval().value,
+    isRefreshPaused:
+      queryService.timefilter.timefilter.getRefreshInterval().pause,
+    onRefreshChange: opts => {
+      queryService.timefilter.timefilter.setRefreshInterval(opts);
```

Comment:
**nitpick**: the interface of `opts` has to be transformed. See the default method to manage this when the `useDefaultBehabiour` is enabled: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.13.0/src/plugins/data/public/ui/search_bar/create_search_bar.tsx#L69-L72

But, I guess this does not update the absolute date range or should it update the date range if the date is relative?